src/main/java/org/folio/rest/impl/InvoiceLineHelper.java

```diff
@@ -365,7 +364,10 @@ public Future<InvoiceLine> createInvoiceLine(InvoiceLine invoiceLine) {
       .compose(v -> createInvoiceLine(ilProcessing, requestContext))
       .compose(v -> orderService.createInvoiceOrderRelation(ilProcessing.getInvoiceLine(), buildRequestContext())
         .recover(throwable -> {
-          throw new HttpException(500, ORDER_INVOICE_RELATION_CREATE_FAILED.toError());
+          String message = String.format(ORDER_INVOICE_RELATION_CREATE_FAILED.getDescription(), throwable.getMessage());
+          var error = ORDER_INVOICE_RELATION_CREATE_FAILED.toError().withMessage(message);
+          logger.error("Failed to create invoice order relation", throwable);
+          throw new HttpException(500, error);
```

Comment:
can you provide screen how it will be shown in browsers Network response, it's basically can be done for any error code, not exactly this one

Reply:
This style has been changed so based on new approach, when api request this error message can be seen
![image](https://github.com/folio-org/mod-invoice/assets/113523904/e7dd9401-60af-4668-add9-7e877f8a9033)